app/models.py

```diff
@@ -347,6 +350,18 @@ class Notification(db.Model):
     status = db.Column(
         db.Enum(*NOTIFICATION_STATUS_TYPES, name='notify_status_types'), nullable=False, default='sending')
     reference = db.Column(db.String, nullable=True, index=True)
+    _personalisation = db.Column(db.String, nullable=True)
```

Comment:
We've left other _ prefixed properties as is for other models, and I don't think it's a egregious situation.